debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -563,6 +565,41 @@ public static OversizeHandlingMode parse(String value, String defaultValue) {
             .withValidation(Field::isBoolean)
             .withDescription("Whether invalid host names are allowed when using SSL. If true the connection will not prevent man-in-the-middle attacks");
 
+    public static final Field SSL_KEYSTORE = Field.create("mongodb.ssl.keystore")
+            .withDisplayName("SSL Keystore")
+            .withType(Type.STRING)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION_ADVANCED_SSL, 1))
+            .withWidth(Width.LONG)
+            .withImportance(Importance.MEDIUM)
+            .withDescription("The location of the key store file. "
+                    + "This is optional and can be used for two-way authentication between the client and the MySQL Server.");
+
+    public static final Field SSL_KEYSTORE_PASSWORD = Field.create("mongodb.ssl.keystore.password")
+            .withDisplayName("SSL Keystore Password")
+            .withType(Type.PASSWORD)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION_ADVANCED_SSL, 2))
+            .withWidth(Width.MEDIUM)
+            .withImportance(Importance.MEDIUM)
+            .withDescription("The password for the key store file. "
+                    + "This is optional and only needed if 'database.ssl.keystore' is configured.");
+
+    public static final Field SSL_TRUSTSTORE = Field.create("mongodb.ssl.truststore")
+            .withDisplayName("SSL Truststore")
+            .withType(Type.STRING)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION_ADVANCED_SSL, 3))
+            .withWidth(Width.LONG)
+            .withImportance(Importance.MEDIUM)
+            .withDescription("The location of the trust store file for the server certificate verification.");
+
+    public static final Field SSL_TRUSTSTORE_PASSWORD = Field.create("mongodb.ssl.truststore.password")
```

Comment:
Did we intentionally prefix these with the `mongodb.ssl.*` namespace rather than just using `ssl.*`?

Reply:
Well, for MySQL these are prefixed "database.ssl.*", additionally MongoDb already contained `mongodb.ssl.enabled`. I see where you are going, however these properties are a bit all over the place across connectors. 

I've talk to @jpechane about potentially restructuring for DBZ 3.0